tests/app/data/test_schema_id_regex.py

```diff
@@ -14,7 +15,7 @@ def create_schema_with_id(schema_id='answer'):
     :param schema_id: The Id to use for the answer.
     :return: The JSON file with the Id swapped for schema_id
     """
-    json_file = open(os.path.join(settings.EQ_SCHEMA_DIRECTORY, "test_percentage.json"))
+    json_file = open(get_schema_file_path("test_percentage.json", language_code='en'))
```

Comment:
We can drop the `language_code` if we default it in `get_schema_file_path` as recommended above